xml/System.Net/SecurityProtocolType.xml

```diff
@@ -29,16 +29,7 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This enumeration defines permissible values for the <xref:System.Net.ServicePointManager.SecurityProtocol%2A> property and specifies the security protocols that are used by instances of the <xref:System.Net.Security.SslStream> class.  
-  
-   
-  
-## Examples  
- The following code example demonstrates creating a <xref:System.Net.Sockets.TcpClient> that uses the <xref:System.Net.Security.SslStream> class to communicate with a server.  
-  
- [!code-cpp[NclSslClientSync#0](~/samples/snippets/cpp/VS_Snippets_Remoting/NclSslClientSync/CPP/clientsync.cpp#0)]
- [!code-csharp[NclSslClientSync#0](~/samples/snippets/csharp/VS_Snippets_Remoting/NclSslClientSync/CS/clientsync.cs#0)]  
-  
+ This enumeration defines permissible values for the <xref:System.Net.ServicePointManager.SecurityProtocol%2A>, which is used by <xref:System.Net.Http.HttpClient> when connecting to a server.  
```

Comment:
Instead of permissible, we should use something a bit easier maybe? Like allowed or acceptable?

Reply:
Sure, that word was there before, and not part of my change. I'll change it, though.

Reply:
I know @stevewhims. But since you changed the sentence, it caught my attention. 😄 Thanks for changing.